src/kit.ts

```diff
@@ -1093,6 +1098,9 @@ export async function scanForKitsIfNeeded(project: CMakeProject): Promise<boolea
  */
 export async function descriptionForKit(kit: Kit, shortVsName: boolean = false): Promise<string> {
     if (kit.toolchainFile) {
+        if (kit.description) {
```

Comment:
This should be moved outside (and above) the branch for `kit.toolchainFile`

Reply:
Thanks for the comment.
Added it to later commit.
Also adjusted kits.md accordingly.